Upgrade dependencies

jraddaoui opened the pull request:
- Use the latest versions of metsrw, ammcpc and agentarchives.
- Upgrade pip-tools to be able to compile those upgrades.

Refs https://github.com/archivematica/Issues/issues/811, https://github.com/archivematica/Issues/issues/812, https://github.com/archivematica/Issues/issues/813.

sevein commented on src/MCPClient/requirements/base.in:
```diff
@@ -1,12 +1,12 @@
--e git+https://github.com/artefactual-labs/ammcpc@v0.1.1#egg=ammcpc
+-e git+https://github.com/artefactual-labs/ammcpc@v0.1.3#egg=ammcpc
```
I published this one a couple of weeks ago, can you use `ammcpc==v0.1.3`?

jraddaoui committed:
Upgrade dependencies

- Use the latest versions of metsrw, ammcpc and agentarchives.
- Upgrade pip-tools to be able to compile those upgrades.

jraddaoui commented:
Thanks @sevein!
